File: MobilCustomer/lib/Models/Customer.dart
class Customer {
  final int id;
  final String? fullName;
  final String? phoneNumber;
  final String? shippingAddress;
  final String? billingAddress;

  Customer({
    required this.id,
    this.fullName,
    this.phoneNumber,
    this.shippingAddress,
    this.billingAddress,
  });

  factory Customer.fromJson(Map<String, dynamic> json) {
    return Customer(
      id: json['id'],
      fullName: json['fullName'],
      phoneNumber: json['phoneNumber'],
      shippingAddress: json['shippingAddress'],
      billingAddress: json['billingAddress'],
    );
  }
}

File: staj/staj/Models/Customer.cs
namespace staj.Models
{
    public class Customer
    {
        public int Id { get; set; }
        public string FullName { get; set; }
        public string PhoneNumber { get; set; }
        public string ShippingAddress { get; set; }  // Teslimat adresi
        public string BillingAddress { get; set; }   // Faturalandırma adresi

        public string Password { get; set; }

        // Mobil tarafındaki müşterinin siparişleri
        public ICollection<Order> Orders { get; set; }
      
    }
}
